From 6fb0b47c65d3b8af1d325dcfc600e598790c38e1 Mon Sep 17 00:00:00 2001
From: Chen Liangjun <b36089@freescale.com>
Date: Thu, 22 Mar 2012 10:54:31 +0800
Subject: [PATCH 288/505] ENGR00177584 ASRC:fix segmentation fault error

when convert from 48k to 96k, segmentation fault may happens.The
issue is caused by wrongly set of data index.

set the data index of output buffer to 0 when start writing the
output data.

Signed-off-by: Chen Liangjun <b36089@freescale.com>
---
 test/mxc_asrc_test/mxc_asrc_test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_asrc_test/mxc_asrc_test.c b/test/mxc_asrc_test/mxc_asrc_test.c
index 5beb6e7..aa4021d 100644
--- a/test/mxc_asrc_test/mxc_asrc_test.c
+++ b/test/mxc_asrc_test/mxc_asrc_test.c
@@ -533,7 +533,7 @@ void convert_data(FILE * dst, struct audio_info_s *info)
 		fwrite(&header[i], 1, 1, dst);
 		i++;
 	}
-
+	i = 0;
 	do {
 		data = output_buffer[i++];
 		data = (data << 8) & 0xFFFF0000;
-- 
1.8.0

